Sources/Build/BuildOperation.swift

```diff
@@ -136,10 +136,14 @@ public final class BuildOperation: PackageStructureDelegate, SPMBuildCore.BuildS
         let oldBuildPath = buildParameters.dataPath.parentDirectory.appending(
             component: buildParameters.configuration.dirname
         )
-        if localFileSystem.exists(oldBuildPath) {
-            try localFileSystem.removeFileTree(oldBuildPath)
+        do {
+            if localFileSystem.exists(oldBuildPath) {
+                try? localFileSystem.removeFileTree(oldBuildPath)
+            }
+            try localFileSystem.createSymbolicLink(oldBuildPath, pointingAt: buildParameters.buildPath, relative: true)
+        } catch {
+            diagnostics.emit(warning: "unable to link \(oldBuildPath): \(error)")
```

Comment:
Actually, one suggestion on the wording here:  in the context of a build, the verb "to link" has connotations of invoking the linker to produce the final executable — could this be changed to something like "unable to create symlink at"?

Reply:
```suggestion
            diagnostics.emit(warning: "unable to create symbolic link at \(oldBuildPath): \(error)")
```